static/app/components/events/breadcrumbs/breadcrumbsDrawer.tsx

```diff
@@ -85,20 +85,15 @@ export function BreadcrumbsDrawer({
     BREADCRUMB_TIME_DISPLAY_LOCALSTORAGE_KEY,
     BreadcrumbTimeDisplay.ABSOLUTE
   );
-  const filterOptions = useMemo(
-    () => getBreadcrumbFilterOptions(breadcrumbs),
-    [breadcrumbs]
-  );
+  const {filterOptions, applyFilters} = useBreadcrumbFilters(breadcrumbs);
```

Comment:
Decided to do it this way since the options should actually be applied in two different ways, though they show up in the same list of filters. 

Rather than having the caller know that `filterOptions` will contain `level-Error` and `type-Exception` and should treat those differently, I just made it spit out a function for applying the filter so all that weird `level-` and `type-` logic is self contained.